bookish/error_handler.py

```diff
@@ -0,0 +1,11 @@
+from flask import jsonify
+from werkzeug.exceptions import HTTPException
+import werkzeug
+
+class CustomHttpException(HTTPException):
+    def __init__(self, e):
+        super().__init__()
+        self.e = e
+def handle_bad_request(e):
+    print(e)
+    return 'bad xczczxrequest!', 403
```

Comment:
`403`  means `Forbidden` not `Bad request` which is `400`

Moreover, seems like the `werkzeug` library includes all the error codes you'd need, I don't think we have to over-engineer a custom error class.